integrations/Rockerbox/index.js

```diff
@@ -1,3 +1,3 @@
-import Rockerbox from './browser';
+import Rockerbox from "./browser";
```

Comment:
```suggestion
import Rockerbox from './browser';
```
The formatter should've taken care of this during the commit. Did you skip the checks?

Reply:
Yes, by mistake, ran it with -n flag though I did delibrately change `singlequote` to `double` as I could see from the last mouse-flow integration PR.
Fixed this.